price/pricing/price_updater_test.go

```diff
@@ -26,7 +26,7 @@ func Test_calculatePrice(t *testing.T) {
 				basePriceUSD: 0.06,
 				multiplier:   1,
 			},
-			want: crypto.FloatToBigMyst(0.1),
```

Comment:
As said comment to `calculatePriceMYST`:
// Take note that this is not 100% correct as we're rounding a bit due to accuracy issues with floats.
// This, however, is not important here as the accuracy will be more than good enough to a few zeroes after the dot.